lib/widgets/footer_section.dart

```diff
@@ -0,0 +1,421 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:festival_flutterturkiye_org/utils/assets.dart';
+import 'package:festival_flutterturkiye_org/widgets/responsive_builder.dart';
+import 'package:material_design_icons_flutter/material_design_icons_flutter.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class FooterSection extends StatelessWidget {
+  const FooterSection({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ResponsiveBuilder(
+      largeWidget: LargeWidget(),
+      mediumWidget: MediumWidget(),
+      smallWidget: SmallWidget(),
+    );
+  }
+}
+
+class SmallWidget extends StatelessWidget {
+  const SmallWidget({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Align(
+        alignment: Alignment.bottomCenter,
+        child: Container(
+          padding: const EdgeInsets.only(top: 20),
+          height: MediaQuery.of(context).size.height / 1.25,
+          width: MediaQuery.of(context).size.width,
+          child: Stack(
+            children: [
+              Container(
+                decoration: BoxDecoration(
+                  image: DecorationImage(
+                    image: AssetImage(Assets.footerBackground),
+                    fit: BoxFit.cover,
+                  ),
+                ),
+              ), // background
+              Container(
```

Comment:
Why did we add this container? It doesn't have any height or width. Is it working?

Reply:
This container added for black filter. It has same height with top widget.